rts/Game/Camera/OverviewController.cpp

```diff
@@ -4,18 +4,25 @@
 #include "OverviewController.h"
 
 #include "Map/Ground.h"
-#include "Map/ReadMap.h"
 #include "Game/UI/MiniMap.h"
 #include "Game/UI/MouseHandler.h"
-#include "Sim/Misc/GlobalConstants.h"
 #include "System/Log/ILog.h"
 
+static float getCamHeightToFitMapInView(float mapx, float mapy, float fov) {
+    static constexpr float marginForUI = 1.037f; // leave some space for UI outside of map edges
+    static constexpr float maxHeight = 25000.0f;
```

Comment:
is this ok? i wasn't sure how to avoid camera going white (from clouds?)

Reply:
Yeah looks ok. I've never seen the code from this class before. I trust it's a good assumption.